### PR TITLE
Test2: Calculate Denominations

### PR DESCRIPTION
**Pull request by Kalaimagal-Prakash**

### CODE REVIEW

File: Training24/Cash.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+   int countCoin[4] = {0, 0, 0, 0};
+
+   for (int i = 0; i < 4; i++) {
+      countCoin[i] = change / denominations[i];
+      change = change % denominations[i];
+   }
+   printf ("10 rupee coins to be returned as: %d coins\n", countCoin[0]);
+   printf ("5  rupee coins to be returned as: %d coins\n", countCoin[1]);
+   printf ("2  rupee coins to be returned as: %d coins\n", countCoin[2]);
+   printf ("1  rupee coins to be returned as: %d coins\n", countCoin[3]);
```
Comment (gdineshkumar219) [inline Training24/Cash.c:24]: `for (int i = 0; i < 4; i++) {
    printf("%d rupee coins to be returned as: %d coins\n", denominations[i], countCoin[i]);
}`
Uses a `for `loop to print the results, significantly reducing repetitive code.

File: Training24/CashTest.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+_CRT_SECURE_NO_WARNINGS;
+#include <stdio.h>
+#include "Cash.h"
+
+int main () {
+   int cashPaid, actualAmount;
+   printf ("Enter cash paid by the customer: ");
+   scanf_s ("%d", &cashPaid);
+   printf ("Enter the actual amount: ");
+   scanf_s ("%d", &actualAmount);
```
Comment (gdineshkumar219) [inline Training24/CashTest.c:18]: Why do you prefer `scanf_s `instead of `fgets`?

**Comment by Dakshita-Minocha** [comment]
Invalid Branch name. Should be: Test2.2

File: Training24/Cash.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+void CalculateChange (int cashPaid, int actualAmount) {
+   int change = cashPaid - actualAmount;
+   int denominations[] = {10, 5, 2, 1};
+   int countCoin[4] = {0, 0, 0, 0};
+
+   for (int i = 0; i < 4; i++) {
+      countCoin[i] = change / denominations[i];
+      change = change % denominations[i];
+   }
+   printf ("10 rupee coins to be returned as: %d coins\n", countCoin[0]);
```
Comment (Dakshita-Minocha) [inline Training24/Cash.c:21]: Library functions should not have printing code.

**Comment by Dakshita-Minocha** [comment]
@Kalaimagal-Prakash You have not written a test.

**Pull request closed without merging**